adding_species_manuscript.tex

```diff
@@ -430,7 +430,7 @@ \section*{Additions to \stdpopsim}
 
 Another important extension of \stdpopsim allows augmenting a genome assembly by genome annotations, such as coding regions, promoters, conserved elements, etc.
 These annotations can be used to simulate selection at a subset of sites (e.g., the annotated coding regions)
-using parametric distribution(s) of fitness effects.
+using a parametric distribution of fitness effects.
```

Comment:
I think that the plural was used here on purpose to indicate that different annotated sets can be associated with different DFEs. Maybe we should use the plural and just remove the parentheses around the s?